### ISSUE TITLE
exec is a reserved word in python 2

### ISSUE THREAD

**Issue by einalex** [opencog/atomspace#2642]
The recent work on python 3 introduced a `exec.pyx/pxd` file in `atomspace/opencog/cython/opencog`
This leads to a syntax error:
```
Traceback (most recent call last):
  File "/home/rasberry/catkin_ws/src/rasberry_hri/scripts/scheduler_node", line 10, in <module>
    from bdi.knowledge_base import *
  File "/home/rasberry/catkin_ws/src/rasberry_hri/src/bdi/knowledge_base.py", line 12, in <module>
    from opencog.bindlink import execute_atom, evaluate_atom
  File "/home/rasberry/git/atomspace/build/opencog/cython/opencog/bindlink.py", line 3
    from opencog.exec import *
                    ^
SyntaxError: invalid syntax
```

`exec` is a reserved word in python: https://docs.python.org/3/library/functions.html#exec

I suggest we rename that file to something not reserved.

**Comment by vsbogd** [comment]
`exec` is not in the list of the Python 3 keywords: https://docs.python.org/3/reference/lexical_analysis.html#keywords And one can define function `exec` and call it. 

But it is a Python 2 keyword https://docs.python.org/2/reference/lexical_analysis.html#keywords
Your issue is more likely related to Python 2/3 mixed environment.

**Comment by einalex** [comment]
well, it's defined in the code, so when I compile it with python 2 and run it, it throws an error.
The only mix of environments here is that the code assumes it isn't run under python 2 and thus uses python 2 reserved words, thus breaking compatibility needlessly.

**Comment by einalex** [comment]
Yes, sorry about that. Unfortunately for this project I'm stuck on py2 :/

**Comment by linas** [comment]
`exec` used to be called `bindlink` and maybe we can keep it that way for python2

**Comment by linas** [comment]
fixed in #2647

**Comment by linas** [comment]
closing, re-open if still an issue
